model/incoming_webhook_test.go

```diff
@@ -101,64 +101,6 @@ func TestIncomingWebhookPreUpdate(t *testing.T) {
 	o.PreUpdate()
 }
 
-func TestIncomingWebhookRequestFromJson_Announcements(t *testing.T) {
```

Comment:
Since the replacement of Slack announcement keys and <@userid> mentions has been moved to `app/slack.go`, this test is removed. Tests have been added for the replacement of the announcement keys and <@userid> mentions in `app/slack_test.go`